docs/developer-guide/printing-module.md

```diff
@@ -0,0 +1,114 @@
+# Printing Module
+
+The **printing module** of MapStore is a back-end service **not included  by default in the release package** that allows to create a printable PDF from the current map.
+
+!!! note
+    The **printing module** is required by the **Print plugin** of MapStore, so if you want to have the Print plugin working in you application, you have to include also the printing module in your MapStore installation.
+
+## Including the printing module in MapStore
+
+Because MapStore doesn't include the printing module by default, to use it you need to build rom the source a `MapStore.war` that includes it or add the missing files to an existing `MapStore.war`
+
+### Building from the Source
+
+If you want to include the printing module in your MapStore, building the source code you have to add the profile `printing` (profiles are the 2nd argument of the `build.sh` script, after the version):
+
+```bash
+./build.sh [version_identifier] printing
+```
+
+Also MapStore projects allow to use the `printing` profile to include this module.
+
+### Adding to an existing MapStore
+
+It is also possible to build only the printing extension as a zip (to be unzipped on your deployed MapStore, inside the `WEB-INF/lib` directory). running `mvn clean install -Pprintingbundle`
+The zip bundle will be in `printing/target/mapstore-printing.zip`.
```

Comment:
i would describe what it means by unzip, because it is not fully clear to me, 
you mean to unzip the whole zip or just the libraries (jar)?

have you checked the versions of spring ?
which the recommended one to use?

maybe you can follow up [this](https://groups.google.com/g/mapstore-developers/c/XZx3hOLrmMU/m/IVouI9npAgAJ) email in the google group

Reply:
you have to unzip the whole zip to the webapps/mapstore folder of your java container